protocol-designer/src/steplist/formLevel/stepFormToArgs/absorbanceReaderFormToArgs.ts

```diff
@@ -20,20 +14,29 @@ export const absorbanceReaderFormToArgs = (
     fileName,
     lidOpen,
     moduleId,
-    // mode,
-    // referenceWavelength,
-    // wavelengths,
+    mode,
+    referenceWavelength,
+    referenceWavelengthActive,
+    wavelengths,
   } = hydratedFormData
   const lidAction = lidOpen
     ? 'absorbanceReaderOpenLid'
     : 'absorbanceReaderCloseLid'
   switch (absorbanceReaderFormType) {
     case ABSORBANCE_READER_INITIALIZE:
+      const rawWavelengths =
+        (mode === 'single' ? wavelengths?.slice(0, 1) ?? null : wavelengths) ?? // only take first wavelength in single mode
```

Comment:
nit, doesn't this do the same thing as slice in this case?
```suggestion
        wavelengths[0]
```

Reply:
I don't think so because we need wavelengths to be an array. I guess I can wrap it in brackets `[wavelengths[0]]`